docs/components/forms/ImageDropZone.tsx

```diff
@@ -0,0 +1,72 @@
+import { useFormikContext } from "formik";
+import React, { useState } from "react";
+import { useDropzone } from "react-dropzone";
+import { ACCEPT_IMAGE_PROP, DropZone } from "../LuxDropZone";
+import { uploadImageToS3 } from "../../utils/upload-file";
+import classNames from "classnames";
+import { getImageUrl } from "../../utils/cdn";
+
+export function ImageDropZone<Values extends Record<string, unknown>>({
```

Comment:
Passing in `FormValues` as a generic is nice 👍 but I think we will want to use a better Dropzone component for this in the future.

Reply:
Yea I think it should be more like `AvatarUploadField` cc @benborgers